src/main/resources/log4j.properties

```diff
@@ -0,0 +1,16 @@
+log4j.rootLogger=INFO, stdout, file
+log4j.logger.org.hibernate=INFO
+log4j.logger.org.hibernate.SQL=DEBUG
+log4j.logger.org.hibernate.type.descriptor.sql=INFO
+
+log4j.appender.stdout=org.apache.log4j.ConsoleAppender
+log4j.appender.stdout.Target=System.out
+log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
+log4j.appender.stdout.layout.ConversionPattern=%d{yyyy-MM-dd HH:mm:ss} %-5p %c{1}:%L - %m%n
+
+log4j.appender.file=org.apache.log4j.RollingFileAppender
+log4j.appender.file.File=C:\\logs\\cinema.log
```

Comment:
let's have this file in `logs` folder that you created in your project

Reply:
Done